contracts/test/plugin/plugin-clonable.ts

```diff
@@ -1,37 +1,78 @@
 import {
-  CloneFactory__factory,
+  DAOMock,
+  DAOMock__factory,
   IPlugin__factory,
   IProtocolVersion__factory,
+  PluginCloneableMockBad__factory,
   PluginCloneableMockBuild1,
   PluginCloneableMockBuild1__factory,
+  ProxyFactory__factory,
 } from '../../typechain';
+import {
+  ProxyCreatedEvent,
+  ProxyFactory,
+} from '../../typechain/src/utils/deployment/ProxyFactory';
 import {erc165ComplianceTests, getOzInitializedSlotValue} from '../helpers';
 import {osxCommonsContractsVersion} from '../utils/versioning/protocol-version';
-import {getInterfaceId, PluginType} from '@aragon/osx-commons-sdk';
+import {
+  findEvent,
+  getInterfaceId,
+  PluginType,
+  PROXY_FACTORY_EVENTS,
+} from '@aragon/osx-commons-sdk';
+import {loadFixture} from '@nomicfoundation/hardhat-network-helpers';
 import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
 import {expect} from 'chai';
 import {ethers} from 'hardhat';
 
-describe('PluginCloneable', function () {
-  let deployer: SignerWithAddress;
-  let plugin: PluginCloneableMockBuild1;
+type FixtureResult = {
+  deployer: SignerWithAddress;
+  implementation: PluginCloneableMockBuild1;
+  proxyFactory: ProxyFactory;
+  daoMock: DAOMock;
+  initCalldata: string;
+  Build1Factory: PluginCloneableMockBuild1__factory;
+};
 
-  before(async () => {
-    [deployer] = await ethers.getSigners();
-    plugin = await new PluginCloneableMockBuild1__factory(deployer).deploy();
-  });
+async function fixture(): Promise<FixtureResult> {
+  const [deployer] = await ethers.getSigners();
+
+  const Build1Factory = new PluginCloneableMockBuild1__factory(deployer);
+  const daoMock = await new DAOMock__factory(deployer).deploy();
+
+  const implementation = await Build1Factory.deploy();
 
+  const proxyFactory = await new ProxyFactory__factory(deployer).deploy(
+    implementation.address
+  );
+
+  const initCalldata = implementation.interface.encodeFunctionData(
+    'initialize',
+    [daoMock.address]
+  );
+
+  return {
+    deployer,
+    implementation,
+    proxyFactory,
+    daoMock,
+    initCalldata,
+    Build1Factory,
+  };
+}
```

Comment:
move to the bottom as done in the tests before. So that the fixtures are always in the same locations. either top or bottom

Reply:
Done in 3d156e7e6accd773f4bc1a0af210556f083dc251